### PR TITLE
Bug: 1883036 - Option to not redeploy the service signer 

### PR DESCRIPTION
**Pull request by vwalek**
The docs for the Bug - 1883036. Adding option to skip the service signer deployment when redeploying certificates.

### CODE REVIEW

File: install/configuring_inventory_file.adoc [hunk 1]
```diff
@@ -207,6 +207,10 @@ signed by an intermediate certificate, provide the bundled certificate that
 contains the full chain of intermediate and root certificates for the CA.
 See xref:../install_config/redeploying_certificates.adoc#redeploying-new-custom-ca[Redeploying a New or Custom {product-title} CA]
 
+|`openshift_redeploy_service_signer`
+| If the parameter is set to `false`, the service signer will not be redeployed during `openshift-master/openshift-redeploy-ca.yml`.
```
Comment (kalexand-rh) [inline install/configuring_inventory_file.adoc:211]:
```suggestion
| If the parameter is set to `false`, the service signer is not redeployed when you run the  `openshift-master/redeploy-certificates.yml` playbook.
```

File: install/configuring_inventory_file.adoc [hunk 2]
```diff
@@ -207,6 +207,10 @@ signed by an intermediate certificate, provide the bundled certificate that
 contains the full chain of intermediate and root certificates for the CA.
 See xref:../install_config/redeploying_certificates.adoc#redeploying-new-custom-ca[Redeploying a New or Custom {product-title} CA]
 
+|`openshift_redeploy_service_signer`
+| If the parameter is set to `false`, the service signer will not be redeployed during `openshift-master/openshift-redeploy-ca.yml`. 
+By default the variable is set to `true`.
```
Comment (kalexand-rh) [inline install/configuring_inventory_file.adoc:212]:
```suggestion
The default value is `true`.
```

File: install_config/redeploying_certificates.adoc [hunk 3]
```diff
@@ -437,6 +437,8 @@ by deleting existing secrets that contain service serving certificates or removi
 annotations to appropriate services.
 ====
 
+You can configure parameter `openshift_redeploy_service_signer=false` in the inventory file to skip the redeployment of service signer certificate if needed.
```
Comment (kalexand-rh) [inline install_config/redeploying_certificates.adoc:440]:
```suggestion
You can set the `openshift_redeploy_service_signer=false` parameter in the inventory file to skip the redeployment of service signer certificate, if required.
```

File: install_config/redeploying_certificates.adoc [hunk 4]
```diff
@@ -437,6 +437,8 @@ by deleting existing secrets that contain service serving certificates or removi
...
+Service Signer certificate is redeployed during the master certificate redeployment when the parameter `openshift_redeploy_openshift_ca=true` is defined.
```
Comment (kalexand-rh) [inline install_config/redeploying_certificates.adoc:441]:
```suggestion
If you set `openshift_redeploy_openshift_ca=true` in the inventory file, the service signing certificate is redeployed when you redeploy the master certificates.
```

**Review by kalexand-rh — COMMENTED** [review]
I have some text edits. @gpei, will you PTAL?

File: install_config/redeploying_certificates.adoc [hunk 5]
```diff
@@ -437,6 +437,8 @@ by deleting existing secrets that contain service serving certificates or removi
 annotations to appropriate services.
 ====
 
+You can set the `openshift_redeploy_service_signer=false` parameter in the inventory file to skip the redeployment of service signer certificate, if required.
+If you set `openshift_redeploy_openshift_ca=true` in the inventory file, the service signing certificate is redeployed when you redeploy the master certificates.
```
Comment (gpei) [inline install_config/redeploying_certificates.adoc:441]: Seen from the code https://github.com/openshift/openshift-ansible/pull/12237/files#diff-f98bfc903bb7557da252f46ccb7f880098e9bd44870e135829704504b479770aR74
Only when openshift_redeploy_openshift_ca=true and openshift_redeploy_service_signer=true(default(true)) , service signing certificate would be redeployed.
If openshift_redeploy_openshift_ca=false or not set, the service signer cert is always not redeployed.

Comment (kalexand-rh) [reply install_config/redeploying_certificates.adoc:441]: @vwalek, will you address this comment and squash?

Comment (vwalek) [reply install_config/redeploying_certificates.adoc:441]: @kalexand-rh  You want to change the text into `Only when openshift_redeploy_openshift_ca=true and openshift_redeploy_service_signer=true(default(true)) , service signing certificate would be redeployed. If openshift_redeploy_openshift_ca=false or not set, the service signer cert is always not redeployed.`

Comment (kalexand-rh) [reply install_config/redeploying_certificates.adoc]: @gpei, will you please take another look?

**Comment by gpei** [comment]
Thanks for the update, it's much more clear now.

File: install_config/redeploying_certificates.adoc [hunk 6]
```diff
@@ -437,6 +437,8 @@ by deleting existing secrets that contain service serving certificates or removi
 annotations to appropriate services.
 ====
 
+You can set the `openshift_redeploy_service_signer=false` parameter in the inventory file to skip the redeployment of service signer certificate, if required.
```
Comment (bobfuru) [inline install_config/redeploying_certificates.adoc:440]: "redeployment **_the_** service signer certificate"?

**Review by bobfuru — COMMENTED** [review]
One small nit; otherwise, LGTM!

**Pull request merged**